### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by conorpp** [knurling-rs/flip-link#34]
There's been substantial improvement over the past 5 months since the last crates.io release.  We're using the latest on master [for SoloKeys](https://github.com/solokeys/solo2) with no issue, but can't make use of the 5 month old release.

Thank you for this awesome tool!

**Comment by nickray** [comment]
To add to this, when using the cargo-install version of flip-link, the addresses according to `arm-none-eabi-size -Ax` don't change, but I guess internally something does, as very weird errors start occurring.

**Comment by Urhengulas** [comment]
> To add to this, when using the cargo-install version of flip-link, the addresses according to `arm-none-eabi-size -Ax` don't change, but I guess internally something does, as very weird errors start occurring.

Hi @nickray, Can you please open a new issue and describe your errors?

**Comment by Urhengulas** [comment]
> There's been substantial improvement over the past 5 months since the last crates.io release. We're using the latest on master [for SoloKeys](https://github.com/solokeys/solo2) with no issue, but can't make use of the 5 month old release.
> 
> Thank you for this awesome tool!

Hi @conorpp, I will check of there is anything we want to pack into a new release, but else I don't see much of a problem doing one in the next days :D

**Comment by nickray** [comment]
> Hi @nickray, Can you please open a new issue and describe your errors?

@Urhengulas I'd rather not; assuming this is a bug that was fixed since the last release. I brought this up here just because I spent about half a day looking elsewhere in code changes for reasons. If the issue persists after a new release (i.e., install from repo works, install from crates.io does not), then I'd revisit. In case you want to investigate anyway, https://github.com/solokeys/solo2/commit/2eded4e566eba3ae3a28282fa439a4c5a0e6fa16 should reproduce.
